pandas/core/indexes/multi.py

```diff
@@ -1847,18 +1847,23 @@ def cats(label):
 
     def sortlevel(self, level=0, ascending=True, sort_remaining=True):
         """
-        Sort MultiIndex at the requested level. The result will respect the
-        original ordering of the associated factor at that level.
+        Sort MultiIndex at the requested level.
+
+        Sort MultiIndex at the requested level and optionally the
+        subsequent ones. The order will respect the internal order of the
+        level elements.
 
         Parameters
         ----------
         level : list-like, int or str, default 0
-            If a string is given, must be a name of the level
+            If a string is given, must be a name of the level.
             If list-like must be names or ints of levels.
         ascending : boolean, default True
-            False to sort in descending order
-            Can also be a list to specify a directed ordering
-        sort_remaining : sort by the remaining levels after level.
+            False to sort in descending order.
+            Can also be a list to specify a directed ordering.
+        sort_remaining : boolean, default True
```

Comment:
Types should be `bool` instead of `boolean`, and `array-like` instead of `list-like`